controllers/scopetemplate_controller.go

```diff
@@ -208,7 +209,9 @@ func (r *ScopeTemplateReconciler) ensureClusterRoles(ctx context.Context, st *op
 		existingCRB.OwnerReferences = clusterRole.OwnerReferences
 		existingCRB.Rules = clusterRole.Rules
 
-		if err := r.Client.Update(ctx, existingCRB); err != nil {
+		// server-side apply patch
+		existingCRB.ManagedFields = nil
+		if err := r.Client.Patch(ctx, existingCRB, client.Apply, client.FieldOwner(scopeTemplateControllerFieldOwner), client.ForceOwnership); err != nil {
```

Comment:
not a blocker but that's one really long call :) Long live 80 character lines!! hehe

Reply:
fixed in commit https://github.com/awgreene/oria-operator/pull/24/commits/adc34c08f252eb9c231f409898d6c84b07678d98 !